inst/EpiModel2.rmd

```diff
@@ -155,6 +164,7 @@ TO standardize the methods for reading and writing to the `dat` object, several
 The `set` group of functions sets the value of an item on a sub-list of the `dat` object; these functions are often used at the end of a modules sequence to update attributes or for tracking of epidemic statistics. As an example: 
```

Comment:
capitalize all headings:

> ### Saving Network and Transmission Matrix

suggestions for line 146: copy-edits

> In parallel with the introduction of `tergmLite` as a dependent package for EpiModel, we updated the output that is saved during simulations. When `tergmLite = FALSE` is passed in as a `control.net` setting, at simulation conclusion the network and transmission matrix are saved to the statistics sub-list of the resultant `netsim` object. Alternatively, if `termLite = TRUE` is input, these objects are not saved. The default is `tergmLite = FALSE`, so the network and transmission matrix are saved as default.

suggestions for lines 150-152: copy-edits

> Prior to EpiModel 2.0, the base functions of EpiModel were built using functions from both `EpiModel` and `network` packages. This intermingling of functions made it difficult for the end user to tease out where network functionality ended and EpiModel began, hampering understanding of the EpiModel workflow and the ability to build module extensions. To more explicitly distinguish the two packages and to make the EpiModel workflow clearer, we introduced a new base function, `nwupdate.net`. This function pulls all interactions with the network object from base functions into `nwupdate.net` and is run at the end of the simulation. By segmenting the EpiModel workflow in this way, base EpiModel modules are now easier to navigate and make clear the output needed for module extensions.
> 
> Note: `nwupdate.net` is a function, like the base initialization function `initialize.net`, that is not meant to be interacted with or changed in most modeling cases. Similarly, because `nwupdate.net` is now handling operations done on network (when `tergmLite` is set to `FALSE`) and networkLite (when `tergmLite` is set to `TRUE`) objects, the user only needs to concern themselves with the core EpiModel disease modules, whether using base EpiModel or designing extension modules.

suggestions for line 156: copy-edits

> New `tergmLite` versions of `network.initialize`, `get.vertex.attribute`, and `set.vertex.attribute` have been updated for use with EpiModel as `network_initialize`, `get_vertex_attribute`, and `set_vertex_attribute`. The default values of each of these functions have been set to run with EpiModel models.

suggestions for line 160: copy-edits and also suggest consistency throughout for `dat` (used below) vs. dat (used above)

> To standardize the methods for reading and writing to the `dat` object, several accessor functions have been added to EpiModel that make clear the indexing and assignment operations that are being done on the master `dat` object during simulation, which means moving away from the `$` and `[[ ]]` operators. These functions belong to the `get` and `set` family of functions and act on the various lists contained within the master `dat` object, including but not limited to the attribute, epi, control, and param lists. These functions take as input, at minimum, the master `dat` object and the item (as a character string) to "get" or "set" (e.g., "age" or "group" in the attribute list or "inf.prob" in the parameter list). Note that these functions and their usage only apply to module extension, where the users are either modifying existing modules or creating their own; they are not meant to be used outside of this setting.